### ISSUE TITLE
International Code of Phytosociological Nomenclature

### ISSUE THREAD

**Issue by mdoering** [SpeciesFileGroup/nomen#20]
Is it in scope to include the International Code of Phytosociological Nomenclature and its governed syntaxon names in NOMEN? They can look an awful lot like Linnean names:

https://www.geobotany.org/library/pubs/WeberHE2000_jvs_739-768.pdf

Examples:
https://de.wikipedia.org/wiki/Rotbuchenwald#Gliederung_der_Rotbuchenwälder
https://lfu.brandenburg.de/cms/media.php/lbm1.a.3310.de/9110.pdf
http://www.natura2000.rlp.de/steckbriefe/index.php?a=s&b=l&pk=9130

**Comment by mdoering** [comment]
https://doi.org/10.2307/3236580

**Comment by mjy** [comment]
IMO any a) formalized and b) international standard for how names for biological entities are governed is a possible inclusion.

**Comment by hlapp** [comment]
That would include the PhyloCode too, no?

**Comment by mdoering** [comment]
... and the BioCode

**Comment by mjy** [comment]
Yes to both.  From my understanding the BioCode in particular is an example of trying to munge everything together, so adding classes for it making them subclasses of existing NOMEN classes may be informative.

Assertions in the PhyloCode are somewhat orthogonal to those in current codes, so in some ways it might be easier to represent concepts there as seperate classes- I think maybe you do this already somewhere in an ontology @hlapp (brain is foggy)?

**Comment by proceps** [comment]
Well, there is also Linz Zoocode, which is an alternative to ICZN.

**Comment by hlapp** [comment]
> Assertions in the PhyloCode are somewhat orthogonal to those in current codes, so in some ways it might be easier to represent concepts there as seperate classes- I think maybe you do this already somewhere in an ontology @hlapp (brain is foggy)?

Yes, we are developing an Ontology of Phylogenetic Clade Definitions, if that's what you mean. It is not meant to be a mirror of PhyloCode names, though; rather a superset.
